### ISSUE TITLE
[Broken Website] www.wasdkeyboards.co

### ISSUE THREAD

**Issue by cipitaua** [darkreader/darkreader#5982]
# Broken Website
## Website
https://www.wasdkeyboards.com/wasd-v3-104-key-custom-mechanical-keyboard.html

## How to Reproduce
- go to link
- click tab "2 Colors"
- keyboard colors are wrongly displayed
- in darkreader switch from  'Dynamic' to 'Static' 
- colors are now ok
- this works must be done every time, it doesn't work if 'Static' is already selected

## Expected behavior
I would expect that static already works at page load

## Actual behavior
'Static' mode works only after switching from 'Dynamic'

## System Information

- OS: 
- Browser: 
- Dark Reader Version:

**Comment by Gusted** [comment]
Hi @cipitaua!

`Doesn't work` is pretty vague in this situation, I've tried it myself and with(out) having first dynamic enable static displays in both situations the same way.

Regards,
Gusted

**Comment by cipitaua** [comment]
Hi @Gusted 
my report states that 'keyboard colors are wrongly displayed'.
This screenshot is with 'Static' mode 

![image](https://user-images.githubusercontent.com/4388859/121245405-49002180-c8a0-11eb-8fec-6da7a9f1bcc9.png)

(note that the color palette on top of the keyboard is totally black)

This is after switching to 'Dynamic' mode:

![image](https://user-images.githubusercontent.com/4388859/121245690-9bd9d900-c8a0-11eb-9150-754905b151c5.png)

(note that white color in the palette is displayed as black)

And this is after switching back to 'Static':

![image](https://user-images.githubusercontent.com/4388859/121245932-dcd1ed80-c8a0-11eb-87f6-a34ff3198b42.png)

Now everything is displayed correctly.

**Comment by cipitaua** [comment]
> Good for you bro wink

yes, but this procedure must be repeated every time i go to the website

**Comment by keronja4** [comment]
Oh

**Comment by cipitaua** [comment]
otherwise I would not have filed a bug report XD

**Comment by cipitaua** [comment]
I have no idea of what you're talking about... but it works :)

thank you and best regards
